process/block/preprocess/smartContractResults.go

```diff
@@ -476,6 +477,43 @@ func (scr *smartContractResults) ProcessMiniBlock(miniBlock *block.MiniBlock, ha
 			&totalGasConsumedInSelfShard)
 
 		if err != nil {
+			//TODO: Remove this after testing
```

Comment:
can't remove this as it is used for the debug print L499

Reply:
This was added to serves the prints and also the prints would been removed now